app/assets/stylesheets/application.css.scss

```diff
@@ -4,4 +4,6 @@
 @import "chosen";
 @import "home";
 @import "users";
+@import "payments";
```

Comment:
not sure why we are using @import here for all.

use the same require pattern here in this file, we can rename this file back to application.css also.
For imports.....

require a file e.g.
 *= require style

in that style file you can @import files if import is required for them e.g.
@import "bootstrap-sprockets";
@import "bootstrap";